src/test/com/microsoft/azure/documentdb/changefeedprocessor/ChangeFeedEventHostTest.java

```diff
@@ -89,7 +109,7 @@ public void createChangeFeedHostUsingSecrets()  {
         Assert.assertNotNull(host);
 
         try {
-            host.registerObserver(TestChangeFeedObserver.class);
+            host.registerObserver(com.microsoft.azure.documentdb.changefeedprocessor.internal.TestChangeFeedObserver.class);

```

Comment:
I think you don't need to make this fully qualified name as you already have imported the package.

Reply:
fixed